site/content/docs/main/deploy-options.md

```diff
@@ -219,6 +219,9 @@ Also see the [upgrade guides][15] on steps to roll out a new version of Contour.
 
 It's possible to run multiple instances of Contour within a single Kubernetes cluster.
 This can be useful for separating external vs. internal ingress, for having separate ingress controllers for different ingress classes, and more.
+Each Contour instance can also be configured via the `--watch-namespaces` flag to handle their own namespaces. This allows the Kubernetes RBAC objects
```

Comment:
A nice follow-up to this work would be to provide a way to set this flag & adjust the RBAC resources via the Gateway provisioner, for folks who are using the provisioner. I'll file a separate issue for that.